ci/pipeline.yml

```diff
@@ -209,6 +217,37 @@ jobs:
         GITHUB_PASSWORD: ((github-password))
         GITHUB_USERNAME: ((github-username))
       image: spring-boot-ci-image
+- name: detect-docker-updates
+  plan:
+  - get: git-repo
+    resource: git-repo-ci-docker
+  - get: every-wednesday
+    trigger: true
+  - get: spring-boot-ci-image
+  - do:
+    - task: detect-docker-updates
+      file: git-repo/ci/tasks/detect-docker-updates.yml
+      params:
+        GITHUB_REPO: spring-boot
+        GITHUB_ORGANIZATION: spring-projects
+        GITHUB_PASSWORD: ((github-password))
+        GITHUB_USERNAME: ((github-username))
+        ISSUE_TITLE: ((docker-upgrade-issue-title))
+      image: spring-boot-ci-image
+    - put: git-repo-ci-docker
+      params:
+        repository: docker-updates-git-repo
```

Comment:
Where is the `docker-updates-git-repo` coming from? I was assuming it needs to be an output in the `detect-docker-updates.yml`?

Reply:
Yes, that's correct. You do have it as an output in `detect-docker-updates.yml` if I'm not wrong.